api/jobs.yaml

```diff
@@ -244,6 +280,46 @@ definitions:
       labels:
         type: object
         description: All job labels, after the update has been applied.
+  AggregationResponse:
+    description: >
+      Returns aggregated jobs with specified time range or label.
+    properties:
+      summary:
+        $ref: '#/definitions/StatusCounts'
+      aggregations:
+        type: array
+        items:
+          $ref: '#/definitions/Aggregation'
+  Aggregation:
+    description: Aggregated jobs specified by the keys.
+    properties:
+      name:
+        type: string
+      key:
+        type: string
+      values:
+        properties:
```

Comment:
This implicitly generates another definition called `AggregationValues`. I'd prefer this to be expressed explicitly so its clearer from looking at the swagger definition what the relevant models are. I would also take the same approach to `StatusCounts` below. It would look something like:

```
  Aggregation:
    description: |
      Aggregation of jobs by label values and statuses, within the specified
      time frame.
    properties:
      name:
        type: string
      key:
        type: string
      entries:
        type: array
        items:
          $ref: '#/definitions/AggregationEntry'
  AggregationEntry:
    description: |
      Aggregation of jobs by statuses for a given label, within a specified time
      frame.
    properties:
      label:
        type: string
        description: The label value for this aggregation entry.
      status_counts:
        $ref: '#/definitions/StatusCounts'
  StatusCounts:
    description: >
      A mapping of JobStatuses to number of jobs with those statuses. If a
      status is not present, its job count is assumed 0.
    properties:
      counts:
        type: array
        items:
          $ref: '#/definitions/StatusCount'
  StatusCount:
    description: A mapping of JobStatus to number of jobs with that status.
    properties:
      status:
        $ref: '#/definitions/JobStatus'
      count:
        type: integer
```